controllers/tasks.js

```diff
@@ -567,12 +579,45 @@ const getUsersHandler = async (req, res) => {
   }
 };
 
+/**
+ * Fetches all the tasks of the logged in user
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+const getTasksByUser = async (req, res) => {
+  try {
+    const { username, id: authenticatedUserId } = req.userData;
+    const requestedUserId = req.params.userId;
+    const dev = req.query.dev;
+
+    if (req.query.version === "v1") {
+      if (requestedUserId === authenticatedUserId && dev) {
```

Comment:
1) use userAuthorization middleware for comparing ID's
2) use devFlagMiddleware for FF.

like this router.post("/example/route", devFlagMiddleware, authenticate, userAuthorization, newTrade, trade);